README.rst

```diff
@@ -161,53 +158,50 @@ Installing multiple versions
 
 On Unix and Mac systems if you intend to install multiple versions of Python
-using the same installation prefix (--prefix argument to the configure script)
+using the same installation prefix (``--prefix`` argument to the configure script)
 you must take care that your primary python executable is not overwritten by the
 installation of a different version.  All files and directories installed using
-"make altinstall" contain the major and minor version and can thus live
-side-by-side.  "make install" also creates ${prefix}/bin/python3 which refers to
-${prefix}/bin/pythonX.Y.  If you intend to install multiple versions using the
+``make altinstall`` contain the major and minor version and can thus live
+side-by-side.  ``make install`` also creates ``${prefix}/bin/python3`` which refers to
+``${prefix}/bin/pythonX.Y``.  If you intend to install multiple versions using the
 same prefix you must decide which version (if any) is your "primary" version.
-Install that version using "make install".  Install all other versions using
-"make altinstall".
+Install that version using ``make install``.  Install all other versions using
+``make altinstall``.
 
 For example, if you want to install Python 2.7, 3.6, and 3.7 with 3.7 being the
-primary version, you would execute "make install" in your 3.7 build directory
-and "make altinstall" in the others.
+primary version, you would execute ``make install`` in your 3.7 build directory
+and ``make altinstall`` in the others.
 
 
 Issue Tracker and Mailing List
 ------------------------------
 
 We're soliciting bug reports about all aspects of the language.  Fixes are also
 welcome, preferably in unified diff format.  Please use the issue tracker:
-
-    https://bugs.python.org/
+`bugs.python.org <https://bugs.python.org/>`_.
 
 If you're not sure whether you're dealing with a bug or a feature, use the
-mailing list:
-
-    python-dev@python.org
-
-To subscribe to the list, use the mailman form:
-
-    https://mail.python.org/mailman/listinfo/python-dev/
+mailing list: python-dev@python.org. To subscribe to the list, use the mailman
+form: `python-dev <https://mail.python.org/mailman/listinfo/python-dev/>`_
 
 
 Proposals for enhancement
 -------------------------
 
 If you have a proposal to change Python, you may want to send an email to the
-comp.lang.python or python-ideas mailing lists for initial feedback.  A Python
+comp.lang.python or
+`python-ideas <https://mail.python.org/mailman/listinfo/python-ideas/>`_
+mailing lists for initial feedback.  A Python
 Enhancement Proposal (PEP) may be submitted if your idea gains ground.  All
 current PEPs, as well as guidelines for submitting a new PEP, are listed at
-https://www.python.org/dev/peps/.
+`python.org/dev/peps/ <https://www.python.org/dev/peps/>`_.
 
 
 Release Schedule
 ----------------
 
-See PEP 494 for release details: https://www.python.org/dev/peps/pep-0494/
+See `PEP 537 <https://www.python.org/dev/peps/pep-0537/>`_ for Python 3.7
```

Comment:
I tried locally using rst2html tool. Sadly, this tool uses the old HTTP link, whereas www.python.org moved to HTTPS. I prefer to use HTTPS.